include/alpaka/mem/buf/oacc/Copy.hpp

```diff
@@ -48,24 +48,29 @@ namespace alpaka
                     typename TTCopyPred>
                 class TTask,
                 typename TDim,
-                typename TViewDst,
+                typename TViewDstFwd,
                 typename TViewSrc,
                 typename TExtent,
                 typename TCopyPred>
             auto makeTaskCopyOacc(
-                TViewDst& viewDst,
+                TViewDstFwd&& viewDst,
                 TViewSrc const& viewSrc,
                 TExtent const& extent,
-                DevOacc const& dev,
+                DevOacc dev,
                 TCopyPred copyPred)
             {
-                return TTask<TDim, TViewDst, TViewSrc, TExtent, TCopyPred>(viewDst, viewSrc, extent, dev, copyPred);
+                return TTask<TDim, std::remove_reference_t<TViewDstFwd>, TViewSrc, TExtent, TCopyPred>(
+                    viewDst,
```

Comment:
I think this call would create a copy even is the initial call was made with a reference. Should this not be forwarded?
```suggestion
                    std::forward<TViewDst>(viewDst),
```

Reply:
Good catch!